dragonchain/webserver/lib/dragonnet.py

```diff
@@ -80,13 +87,51 @@ def process_receipt_v1(block_dto: Dict[str, Any]) -> None:
                 _log.exception("matchmaking add_receipt failed!")
             # Update the broadcast system about this receipt
             broadcast_functions.set_receieved_verification_for_block_from_chain_sync(l1_block_id, level_received_from, block_model.dc_id)
+            # Send verifications to any configured urls
+            send_verification_notifications(level_received_from, block_model)
         else:
             _log.warning(
                 f"Chain {block_model.dc_id} (level {level_received_from}) returned a receipt that wasn't expected (possibly expired?) for block {l1_block_id}. Rejecting receipt"  # noqa: B950
             )
             raise exceptions.NotAcceptingVerifications(f"Not accepting verifications for block {l1_block_id} from {block_model.dc_id}")
 
 
+def get_notification_urls(key: str) -> set:
+    urls = None
+    try:
+        urls = VERIFICATION_NOTIFICATION[key]
+    except KeyError:
+        pass
```

Comment:
Could change this to `urls = set()` and then you can get rid of line 100, 105, 106

Reply:
done